### PR TITLE
Added a  link to a YouTube video on Azure Secret Store Provider for AKV integration with VMSS Managed Identity

### PR DESCRIPTION
**Pull request by NileshGule**
**What this PR does / why we need it**:
Shows an end to end example of integrating AKV secrets with VMSS Managed Identity based AKS cluster. The AKV secrets are synched to Kubernetes secret and injected as ENV variables into a deployment for a RabbitMQ consumer.

### CODE REVIEW

**Comment by NileshGule** [comment]
@aramase the PR has been rebased. Please help to merge

File: README.md [hunk 1]
```diff
@@ -250,10 +254,14 @@ For documentation on how to locally test the Secrets Store CSI Driver Provider f
...
 
 ## Presentations
 
 This demo created by <a href="https://twitter.com/HoussemDellai">Houssem Dellai</a> is using AAD Pod Identity and Secret Store CSI provider for Key Vault to retrieve database login and password from Azure Key Vault. <a href="https://www.youtube.com/watch?v=dAFWrbeA6vQ&list=PLpbcUe4chE79sB7Jg7B4z3HytqUUEwcNE&index=24">Watch it here.</a>
 
 <a href="https://www.youtube.com/watch?v=dAFWrbeA6vQ&list=PLpbcUe4chE79sB7Jg7B4z3HytqUUEwcNE&index=24"><img src="https://github.com/HoussemDellai/secrets-store-csi-driver-provider-azure/blob/added-demo/images/demo-video-thumb.png?raw=true" width="60%"/></a>
+
+This demo created by <a href="https://twitter.com/NileshGule">Nilesh Gule</a> is using VMSS Managed Identity and Secret Store CSI provider for Key Vault to retrieve RabbitMQ related secret from Azure Key Vault. These secrets are synched with Kubernetes Secret object and then injected into deployment as ENV variables. <a href="https://youtu.be/MHm4IVGVO1w">Integrate AKV with AKS in 5 easy steps</a>
```
Comment (aramase) [inline README.md:265]: nit: s/synched/synced

Comment (NileshGule) [reply README.md]: hi @aramase this has been fixed

**Comment by aramase** [comment]
@NileshGule One last rebase and then we can get this merged.

**Comment by NileshGule** [comment]
@aramase the PR has been updated

File: README.md [hunk 2]
```diff
@@ -257,10 +262,14 @@ For documentation on how to locally test the Secrets Store CSI Driver Provider f
...
 
 ## Presentations
 
 This demo created by <a href="https://twitter.com/HoussemDellai">Houssem Dellai</a> is using AAD Pod Identity and Secret Store CSI provider for Key Vault to retrieve database login and password from Azure Key Vault. <a href="https://www.youtube.com/watch?v=dAFWrbeA6vQ&list=PLpbcUe4chE79sB7Jg7B4z3HytqUUEwcNE&index=24">Watch it here.</a>
 
-<a href="https://www.youtube.com/watch?v=dAFWrbeA6vQ&list=PLpbcUe4chE79sB7Jg7B4z3HytqUUEwcNE&index=24"><img src="https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/images/demo-video-thumb.png?raw=true" width="60%"/></a>
+<a href="https://www.youtube.com/watch?v=dAFWrbeA6vQ&list=PLpbcUe4chE79sB7Jg7B4z3HytqUUEwcNE&index=24"><img src="https://github.com/HoussemDellai/secrets-store-csi-driver-provider-azure/blob/added-demo/images/demo-video-thumb.png?raw=true" width="60%"/></a>
```
Comment (aramase) [inline README.md:271]:
```
<a href="https://www.youtube.com/watch?v=dAFWrbeA6vQ&list=PLpbcUe4chE79sB7Jg7B4z3HytqUUEwcNE&index=24"><img src="https://github.com/HoussemDellai/secrets-store-csi-driver-provider-azure/blob/added-demo/images/demo-video-thumb.png?raw=true" width="60%"/></a>
```

@NileshGule Could you revert this change? The URL is fixed in the master and this change is adding back the old URL.

Comment (aramase) [reply README.md:271]: ping @NileshGule to resolve the outstanding comment

Comment (NileshGule) [reply README.md]: @aramase I have updated the link to the image. Please review the latest changes.

**Comment by aramase** [comment]
@NileshGule Looks like with the last commit, the changes aren't rebased. Could you rebase this PR and also squash the commits?

**Comment by aramase** [comment]
@NileshGule Ping! Could you rebase the PR?

**Comment by NileshGule** [comment]
hi @aramase 
I have rebased the PR

Not sure why the Readme is showing a conflict. I cannot find this conflict in my repository :(

**Comment by aramase** [comment]
@NileshGule There is one merge conflict here -

```
<<<<<<< master
You can setup the Secrets Store CSI Driver to periodically update the pod mount and Kubernetes Secret with the latest content from external secrets-store. Refer to [doc](<[docs/README.rotation.md](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/docs/README.rotation.md)>) for steps on enabling auto rotation.
=======
You can setup the Secrets Store CSI Driver to periodically update the pod mount and Kubernetes Secret with the latest content from external secrets-store. Refer to [doc](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/docs/README.rotation.md) for steps on enabling auto rotation.
>>>>>>> master
```

**Comment by NileshGule** [comment]
hi @aramase 
I have fixed the conflict. please help to merge

**Review by aramase — APPROVED** [review]
lgtm
